main.cpp

```diff
@@ -66,7 +97,7 @@ static llvm::cl::OptionCategory CastMatcherCategory("cast-matcher options");
 
 int main(int argc, const char **argv) {
     // Ubuntu 20
-    auto Parser = llvm::ExitOnError()(CommonOptionsParser::create(argc, argv, CastMatcherCategory));
+    auto Parser = llvm::ExitOnError()(CommonOptionsParser::create(argc, argv, CastMatcherCategory, llvm::cl::OneOrMore)); // worked for me

```

Comment:
The `llvm::cl::OneOrMore` value for `OccurrencesFlag` is passed by default. Could you explain please why the version without passing `OccurrencesFlag` does not work?

Reply:
In my case, I have this error. (prebuild clang-15+llvm,virtual machine with Ubuntu 20.04)
![image](https://user-images.githubusercontent.com/37217930/170352595-9e026141-9541-4a81-9f08-0bba9659b231.png)